perfil/person/models.py

```diff
@@ -30,10 +30,38 @@ def election_parties(self):
         parties = [x[0] for x in self.elections.values_list('party__initials')]
         return list(set(parties))
 
+    @property
+    def filiation_parties(self):
+        parties = [x[0] for x in self.filiations.values_list('party__initials')]
+        return list(set(parties))
```

Comment:
Can we use generators and tuples instead of lists? This saves a bit of memory, tiny bits…

```python
parties = (x[0] for x in self.filiations.values_list('party__initials'))
return tuple(set(parties))
```